brokerapi/brokers/config/broker_config.go

```diff
@@ -0,0 +1,105 @@
+// Copyright the Service Broker Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+//
+
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"gcp-service-broker/brokerapi/brokers/models"
+	"gcp-service-broker/utils"
+	"golang.org/x/oauth2/jwt"
+	"gopkg.in/validator.v2"
+	"os"
+)
+
+type BrokerConfig struct {
+	Catalog    map[string]models.Service
```

Comment:
Do you need to store this as a map here? If not, you could store it as a list and then you won't need a public ServiceList method.